### PR TITLE
fix(sql): support comment-only query

### PR DESCRIPTION
**Pull request by mtopolnik**
QuestDB currently supports a completely empty query, but not an empty query that has a comment. This PR fixes it.

This PR does not fix a multi-statement script that ends in a comment. This still fails in extended mode, but with a different error ("Received resultset tuples, but no field structure for them"). This problem should be addressed with the upcoming PGWire 2.0.

### CODE REVIEW

File: core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java [hunk 1]
```diff
@@ -128,6 +128,7 @@ public JsonQueryProcessor(
             this.queryExecutors.extendAndSet(CompiledQuery.CREATE_USER, sendConfirmation);
             this.queryExecutors.extendAndSet(CompiledQuery.ALTER_USER, sendConfirmation);
             this.queryExecutors.extendAndSet(CompiledQuery.CANCEL_QUERY, sendConfirmation);
+            this.queryExecutors.extendAndSet(CompiledQuery.EMPTY, sendConfirmation);
```
Comment (ideoma) [inline core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java:131]: Please add a test with empty query that uses this context, e.g. in `IODispatcherTest`

Comment (mtopolnik) [reply core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java]: `sendConfirmation` sends `{"ddl": "OK"}`, which isn't appropriate for this (as well as many other cases that use it, like `COMMIT`, `ROLLBACK` and more). I changed it to respond with 200 OK, but JSON says `{"error": "empty query" ...}`.

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 26 / 26 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|15|15|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|

**Review by ideoma — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 26 / 26 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|15|15|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|2|2|100.00%|
